test/parallel/test-worker-stdio.js

```diff
@@ -27,7 +27,7 @@ if (isMainThread) {
   const passed = new BufferingWritable();
 
   const w = new Worker(__filename, { stdin: true, stdout: true });
-  const source = fs.createReadStream(process.execPath);
+  const source = fs.createReadStream(process.execPath, { end: 1000000 });
```

Comment:
nit: maybe make it 1MiB instead:
```suggestion
  const source = fs.createReadStream(process.execPath, { end: 1 << 10 << 10 });
```

Alternatively, use number separator to improve readability:
```suggestion
  const source = fs.createReadStream(process.execPath, { end: 1_000_000 });
```